apps/next-app/src/pages/_app.tsx

```diff
@@ -0,0 +1,7 @@
+import '../styles/globals.css';
+import type { AppProps } from 'next/app';
+
+export default function App({ Component, pageProps }: AppProps) {
+  // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
why this comment?

Reply:
it was because of previous linting setup removed now